src/server/handlers/userHelpers.js

```diff
@@ -0,0 +1,13 @@
+import { User } from '#schemas/users/userSchema.js';
+
+async function getUserById(id) {
+  const user = await User.findById(id);
+  return user;
+}
+
+async function getUserByEmail(email) {
+  const user = await User.findOne({ email });
+  return user;
+}
+
+export { getUserByEmail, getUserById };
```

Comment:
Would be nice to have `getUserByVerifyToken` because I think we will reuse in two cases. 
We will be able to use it in simple way like so passing only token.

Reply:
❓